environment.yml

```diff
@@ -21,6 +21,7 @@ dependencies:
   - bioconda::genesplicer=1.0
   - bioconda::htslib=1.9
   - bioconda::manta=1.6.0
+  - bioconda::msisensor=0.5
```

Comment:
Could you update the `docs/containers.md` as well please?

Reply:
And while your at it, could you please also add a line in the CHANGELOG about adding msisensor:
https://github.com/nf-core/sarek/blob/dev/CHANGELOG.md

Reply:
done in 41154f551aacaf2ddec2797b70839b0aad087dd4